index.js

```diff
@@ -26,11 +24,7 @@ var main = function(url, options) {
 	try {
 		options = options || {};
 
-		if (typeof options !== "object") {
-			throw "Expected an object";
-		}
-
-		return options.embed === true ? getEmbed(url) : getPanoID(url);
+		return options.embed ? getEmbed(url) : getPanoID(url);
```

Comment:
It's all right.
I just made some improvements. Nothing ~really~ special.
Closing this pull request for now. 👍